src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -80,7 +80,7 @@ class IntroductionPage extends React.Component {
           pageList={this.props.route.pageList}
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}>
-          Complete the form before submitting to apply for pension benefits with the 21-527EZ.
+          Please complete the 21-527EZ form to apply for pension benefits.
```

Comment:
@jbalboni is it alright to update this to match the other introduction pages?

Reply:
Yep, go for it.

Reply:
Also, can you update the 5495 one to have the `22-` in front of the form number?